examples/webpack/karma.conf.js

```diff
@@ -0,0 +1,88 @@
+const webpack = require('webpack');
+
+const webpackConfig = {
+  module: {
+    loaders: [{
+      test: /\.ts$/,
+      loader: 'ts',
+      exclude: /node_modules/
+    }],
+    postLoaders: [{
+      test: /src\/.+\.ts$/,
+      exclude: /(test|node_modules|\.spec)/,
```

Comment:
I think that would work better. Do you have time to make that change, I think we can merge once this is done.